Gemfile

```diff
@@ -72,3 +72,7 @@ group :test do
 end
 
 gem "devise", "~> 4.8"
+
+gem "yt", "~> 0.33.4"
+
+gem "figaro", "~> 1.2"
```

Comment:
can please explain a little bit about what gem is doing.

Reply:
1. yt gem is to enable connection with youtube (i think it could still work without it but i haven't checked yet)
2. figaro gem is just to store all the environmental variables. (i created an api key for yt gem)

Reply:
Yes! i also think that, there will be no require for yt gem(if it is using only for enable connection with youtube).